### PR TITLE
Create TLC modal

### PR DESCRIPTION
**Pull request by nichosystem**
# Description
- Add sepolia
- Swap wagmi for web3-modal
- Add TLC modal, nested under /tlc route

# Checklist
- [x] Code follows the style guide
- [x] I have performed a self-review of my own code
- [x] New and existing tests pass locally
- [x] This PR is targeting the correct branch

### CODE REVIEW

**Comment by nichosystem** [comment]
Deploy preview for _core_ ready!

✅ Preview
https://core-arr440zc7-templedao.vercel.app
https://pr-825-preview.templedao.link

Built with commit 48acfaccc5aa144200a99bb9f43c728eb052095c.
This pull request is being automatically deployed with [vercel-action](https://github.com/marketplace/actions/vercel-action)

**Review by marshall2112 — CHANGES_REQUESTED** [review]
A couple small questions. Manually tested everything in the new modal it works as expected.

File: apps/dapp/src/constants/env/preview.tsx [hunk 1]
```diff
@@ -2,37 +2,38 @@ import { ADDRESS_ZERO } from 'utils/bigNumber';
...
-    faith: '0x2c20342F1B27Ca1E4e6668A623084Bb9fC086A4D',
-    farmingWallet: '0x5C8898f8E0F9468D4A677887bC03EE2659321012',
-    frax: '0x73651AD693531F9937528009cC204a4d9b696a68',
-    usdc: '0x07865c6e87b9f70255377e024ace6630c1eaa37f',
+    balancerVault: '',
+    exitQueue: '',
+    faith: '',
+    farmingWallet: '',
+    frax: '',
+    usdc: '',
```
[the remark this answers is not in the document]
Comment (nichosystem) [reply apps/dapp/src/constants/env/preview.tsx:14]: balancer wasn't available on testnet for a while but they did recently deploy to sepolia; so for now we're alright as it's status quo to not test swaps on testnet atm, but I will add it to the todo to support balancer sepolia in the future

File: apps/dapp/src/components/Pages/Core/NewUI/TLC/Borrow.tsx [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+          {tlcInfo ? (tlcInfo.borrowRate * 100).toFixed(2) : 0}% <span>interest rate</span>
+        </Apy>
+        <Rule />
+        <Copy>{liquidationInfo(Number(state.borrowValue))}</Copy>
+      </GradientContainer>
+      <RiskAcknowledgement>
+        <Checkbox onClick={() => setCheckbox(!checkbox)} isChecked={checkbox} src={checkmark} />
+        <div>
+          <p>I acknowledge the risks of borrowing including increased risk of liquidation.</p>
+          <a href="#">Find out more</a>
```
[the remark this answers is not in the document]
Comment (nichosystem) [reply apps/dapp/src/components/Pages/Core/NewUI/TLC/Borrow.tsx:156]: yeah, we don't have TLC documentation written atm though. I'll comment this out for now

**Review by marshall2112 — APPROVED** [review]
LGTM

**Pull request merged**